Sources/Parsers/AssetsCatalogParser.swift

```diff
@@ -104,6 +107,9 @@ extension AssetsCatalogParser {
     let type = item.extension ?? ""
 
     switch (type, AssetCatalog.Item.supported.contains(type)) {
```

Comment:
👌 good catch in that documentation, I forgot they published that! (Worth putting a link to it as a code comment at the top of the XCAssetsParser if not already done) 

Let's not allow periods in group names and follow the documentation, option 2 it is!